cardano-node/src/Cardano/CLI/Tx/Generation.hs

```diff
@@ -390,6 +393,7 @@ prepareInitialFunds
   -> CC.Common.Address
   -> CC.Common.Address
   -> FeePerTx
+  -> TVar.TVar AvailableFunds
   -> IO ()
```

Comment:
..and as mentioned above, this could become a return value, instead of a mutable variable.